contrib/import-control.xml

```diff
@@ -37,18 +37,6 @@
     <allow pkg="org.apache.accumulo.core.conf"/>
     <allow class="org.apache.accumulo.core.util.threads.ThreadPools"/>
 
-    <!-- TODO refactor code to remove the following exceptions -->
-    <allow class="org.apache.accumulo.core.metadata.MetadataTable"/>
-    <allow class="org.apache.accumulo.core.replication.ReplicationTable"/>
-    <allow class="org.apache.accumulo.core.spi.scan.HintScanPrioritizer"/>
-    <allow class="org.apache.accumulo.core.clientImpl.TabletServerBatchWriter"/>
-    <allow class="org.apache.accumulo.core.util.FastFormat"/>
-    <allow class="org.apache.accumulo.core.util.Pair"/>
-    <allow class="org.apache.accumulo.core.trace.Trace"/>
```

Comment:
As is, there are commented out usages of Trace in [VerifyIngest.java](https://github.com/apache/accumulo-testing/blob/fbaf16dc2c065eaab690d8d7abd75750d5d0425c/src/main/java/org/apache/accumulo/testing/ingest/VerifyIngest.java#L65). I'm thinking that either the comments should be removed along with the `<allow/>` tag or, both should be kept. For now I'll remove both and change it back if it seems like it should be.

Reply:
My guess is that was commented out because either it wasn't working or the Trace code was in flux (it still is). Since there is an option to use the Trace in VerifyIngest, we could throw an exception stating it is currently not supported.

Reply:
Something like 96ff1ac? @milleruntime